arangod/RestHandler/RestAdminClusterHandler.cpp

```diff
@@ -2710,6 +2710,31 @@ RestAdminClusterHandler::collectRebalanceInformation(
   cluster::rebalance::AutoRebalanceProblem p;
   p.zones.emplace_back(cluster::rebalance::Zone{.id = "ZONE"});
 
+  std::string const leaderPath = "Plan/AsyncReplication/Leader";
```

Comment:
This seems superfluous. This key is something for active failover, not needed here.